lib/pharos/kube/session.rb

```diff
@@ -20,10 +20,11 @@ def resource(resource)
 
       # Returns a new stack associated with this session
       # @param name [String]
+      # @param resource_path [String]
       # @param vars [Hash]
       # @return [Pharos::Kube::Stack]
-      def stack(name, vars = {})
-        Stack.new(self, name, vars)
+      def stack(name, resource_path, vars = {})
+        Stack.new(self, name, resource_path, vars)
```

Comment:
If the resource path is required, I think the name could be derived from `File.basename(resource_path)`

The name I think was mostly used for finding the subdir under `resources/`

Reply:
Not sure if we can make it work.. path points to a sub-folder `resources`.

Reply:
Right. I guess it could then be just stack path and the resource_path would be that + resources/